cilium/error.go

```diff
@@ -194,3 +211,30 @@ func ErrLoadNamespace(err error, s string) error {
 	return errors.New(ErrLoadNamespaceCode, errors.Alert, []string{"Error occured while applying namespace "}, []string{err.Error()}, []string{"Trying to access a namespace which is not available"}, []string{"Verify presence of namespace. Confirm Meshery ServiceAccount permissions"})
 
 }
+
+// ErrMakingBinExecutable occurs when cilium cli binary couldn't be made
+// executable
+func ErrMakingBinExecutable(err error) error {
+	return errors.New(ErrMakingBinExecutableCode, errors.Alert, []string{"Error while making cilium cli an executable"}, []string{err.Error()}, []string{}, []string{})
+}
+
+// ErrUnpackingTar is the error when tar unpack fails
+func ErrUnpackingTar(err error) error {
+	return errors.New(ErrUnpackingTarCode, errors.Alert, []string{"Error occured while unpacking tar"}, []string{err.Error()}, []string{}, []string{})
+}
+
+// ErrUnzipFile is the error for unzipping the file
+func ErrUnzipFile(err error) error {
+	return errors.New(ErrUnzipFileCode, errors.Alert, []string{"Error while unzipping"}, []string{err.Error()}, []string{"File might be corrupt"}, []string{})
```

Comment:
Let’s be sure to have the probable cause and suggested remediation filled in for each of these new error codes.